go.mod

```diff
@@ -10,9 +10,11 @@ require (
 	github.com/edgexfoundry/go-mod-messaging/v3 v3.2.0-dev.31
 	github.com/edgexfoundry/go-mod-registry/v3 v3.2.0-dev.13
 	github.com/fxamacker/cbor/v2 v2.7.0
-	github.com/gomodule/redigo v1.8.9
+	github.com/gomodule/redigo v2.0.0+incompatible
```

Comment:
is it a necessary upgrade?

Reply:
No, it just upgraded automatically after running `go mod tidy`. I'll revert this back.